parkings/tests/api/operator/test_disc_parking.py

```diff
@@ -45,6 +47,7 @@ def test_post_disc_parking(operator_api_client, operator, disc_parking_data):
 
 
 def test_end_disc_parking_with_patch(operator_api_client, operator, disc_parking):
+    create_enforced_domain_for_user(operator.user)
```

Comment:
Same here. Is this needed? https://github.com/City-of-Helsinki/parkkihubi/pull/146/commits/d2cd37559501da7fbec0130bfe7d4f9ad860b437#r412981973